dbildungs-iam-server-secret.template.yaml

```diff
@@ -5,7 +5,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: dbildungs-iam-secrets
+  name: dbildungs-iam-server
```

Comment:
I think this file should get deleted, it is not up to date and not needed anymore, because there is a secret definition file in the helm chart.

Reply:
Thanks! Deleted it :)